protocol/x/clob/keeper/process_operations.go

```diff
@@ -144,8 +144,15 @@ func (k Keeper) ProcessInternalOperations(
 
 		switch castedOperation := operation.Operation.(type) {
 		case *types.InternalOperation_Match:
+			affiliatesWhitelistMap, err := k.affiliatesKeeper.GetAffiliateWhitelistMap(ctx)
```

Comment:
Declare `affiliatesWhitelistMap` before the for loop, and query it at most once for all matches in the operations queue.